front-end/src/app/shared/components/transaction-type-base/transaction-contact.utils.ts

```diff
@@ -145,4 +145,38 @@ export class TransactionContactUtils {
       }
     }
   }
+
+  static updateContactFromForm(form: FormGroup, transaction: Transaction) {
```

Comment:
it seems like this is fulfilling the same purpose as setTransactionContactFormChanges().  Are we doubling up our code?

Reply:
yes, the old function did the trick. am removing the new one